docs/reinstallation/README.md

```diff
@@ -0,0 +1,383 @@
+3. [Initiate a cluster reinstallation.](#3-initiating-a-cluster-reinstallation)
+4. [Optional. Monitor the reinstallation progress.](#4-optional-monitoring-the-cluster-reinstallation)
+5. [Verify that the cluster is provisioned and available.](#5-verify-cluster-availability)
+
+You can complete step 1. and 2. well before of initiating the cluster reinstallation.
+
+### 1. Enabling cluster reinstallation in the SiteConfig operator
+
+You must explicitly enable cluster reinstallation in the `siteconfig-operator-configuration` `ConfigMap` resource, which you can complete well before initiating the process.
+
+#### Checking the current configuration  
+By default, reinstallation is disabled. You can check the current configuration by completing the following steps:
+
+Create the `NAMESPACE` environment variable for the `siteconfig-operator-configuration` `ConfigMap` resource. Run the following command:
+
+```sh
+NAMESPACE=<namespace>
+```
+
+Verify the current configuration by running the following command:
+
+```sh
+oc get configmap siteconfig-operator-configuration -n $NAMESPACE -o yaml
+```
+
+```yaml
+apiVersion: v1  
+kind: `ConfigMap`  
+metadata:  
+  name: siteconfig-operator-configuration  
+  namespace: <namespace>  
+data:  
+  allowsReinstalls: false  
+  ...  
+```
+
+- `<namespace>`: The namespace where the SiteConfig operator is installed.  
+- `allowsReinstalls`: Set to `true` enables reinstallation. 
+
+The operator continuously monitors `siteconfig-operator-configuration` `ConfigMap` resource for changes.
+
+#### Enabling cluster reinstallation  
+
+Create the `NAMESPACE` environment variable for the `siteconfig-operator-configuration` `ConfigMap` resource. Run the following command:
+
+```sh
+NAMESPACE=<namespace>
+```
+
+Update the `ConfigMap` resource by running the following command:
+
+```sh
+oc patch configmap siteconfig-operator-configuration \  
+  -n $NAMESPACE \  
+  --type=json \  
+  -p '[{"op": "replace", "path": "/data/allowsReinstalls", "value": "true"}]'  
+```
+
+Verify the update by running the following command:
+
+```sh
+oc get configmap siteconfig-operator-configuration -n $NAMESPACE -o yaml  
+```
+
+
+### 2. Labeling resources for preservation
+
+To retain essential cluster configuration data after reinstallation, the SiteConfig operator provides a backup and restore mechanism for `ConfigMap` and `Secret` resources within the `ClusterInstance` namespace.
+
+To preserve specific resources, apply the appropriate preservation label as described in the [preservation document](#preservation.md). You can label your resources well before initiating the reinstallation.
+
+The SiteConfig operator backs up resources based on the following labels:
+
+- `siteconfig.open-cluster-management.io/preserve: "<arbitrary-value>"`: Backs up both non-identity-related and cluster identity-related resources.
+- `siteconfig.open-cluster-management.io/preserve: "cluster-identity"`: Backs up cluster identity-related resources.
```

Comment:
CRs with this label will be backed up under either the All or ClusterIdentity preservation Mode